src/msbuild/DNNE.targets

```diff
@@ -87,6 +87,11 @@ DNNE.targets
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <Visible>false</Visible>
     </None>
+    <None Include="$(DnneGeneratedBinPath)/$(DnneNativeExportsBinaryName).pdb">
```

Comment:
This isn't going to work for non-Windows platforms. Instead, see line 165 below for Windows specific file copy.